src/Templates/AdminPanel/Bit.AdminPanel/Client/Shared/Pages/Products/CreateEditProductModal.razor.cs

```diff
@@ -6,6 +6,8 @@ public partial class CreateEditProductModal
 {
     [Parameter]
     public ProductDto Product { get; set; }
+    [Parameter]
+    public EventCallback<MouseEventArgs> OnSave { get; set; }
```

Comment:
please also remove the argument (MouseEventArgs) from the EventCallback like this.

`public EventCallback OnSave { get; set; }`